libmscore/hairpin.cpp

```diff
@@ -408,6 +408,13 @@ void Hairpin::read(XmlReader& e)
             else if (!TextLine::readProperties(e))
                   e.unknown();
             }
+
+      // add default text to legacy hairpins
+      if (score()->mscVersion() <= 206) {
```

Comment:
Sorry I should have reviewed this before but I think it would be good to no override the existing text if any. Like if we have a 2.0.3 version with 2.06 fileformat but with cresc line support.